projects/1.20-fabric/assets/more-slabs-stairs-and-walls/more_slabs_stairs_and_walls/lang/zh_cn.json

```
@@ -0,0 +1,831 @@
+  "block.more_slabs_stairs_and_walls.chiseled_quartz_block_slab": "雕纹石英台阶",
+  "block.more_slabs_stairs_and_walls.chiseled_quartz_block_stairs": "雕纹石英楼梯",
+  "block.more_slabs_stairs_and_walls.chiseled_quartz_block_wall": "雕纹石英墙",
+  "block.more_slabs_stairs_and_walls.chiseled_red_sandstone_slab": "雕纹红砂岩台阶",
+  "block.more_slabs_stairs_and_walls.chiseled_red_sandstone_stairs": "雕纹红砂岩楼梯",
+  "block.more_slabs_stairs_and_walls.chiseled_red_sandstone_wall": "雕纹红砂岩墙",
+  "block.more_slabs_stairs_and_walls.chiseled_sandstone_slab": "雕纹砂岩台阶",
+  "block.more_slabs_stairs_and_walls.chiseled_sandstone_stairs": "雕纹砂岩楼梯",
+  "block.more_slabs_stairs_and_walls.chiseled_sandstone_wall": "雕纹砂岩墙",
+  "block.more_slabs_stairs_and_walls.chiseled_stone_bricks_slab": "雕纹石砖台阶",
+  "block.more_slabs_stairs_and_walls.chiseled_stone_bricks_stairs": "雕纹石砖楼梯",
+  "block.more_slabs_stairs_and_walls.chiseled_stone_bricks_wall": "雕纹石砖墙",
+  "block.more_slabs_stairs_and_walls.clay_slab": "黏土台阶",
+  "block.more_slabs_stairs_and_walls.clay_stairs": "黏土楼梯",
+  "block.more_slabs_stairs_and_walls.clay_wall": "黏土墙",
+  "block.more_slabs_stairs_and_walls.coal_block_slab": "煤炭台阶",
+  "block.more_slabs_stairs_and_walls.coal_block_stairs": "煤炭楼梯",
+  "block.more_slabs_stairs_and_walls.coal_block_wall": "煤炭墙",
+  "block.more_slabs_stairs_and_walls.coarse_dirt_slab": "砂土台阶",
+  "block.more_slabs_stairs_and_walls.coarse_dirt_stairs": "砂土楼梯",
+  "block.more_slabs_stairs_and_walls.coarse_dirt_wall": "砂土墙",
+  "block.more_slabs_stairs_and_walls.copper_block_slab": "铜台阶",
+  "block.more_slabs_stairs_and_walls.copper_block_stairs": "铜楼梯",
+  "block.more_slabs_stairs_and_walls.copper_block_wall": "铜墙",
+  "block.more_slabs_stairs_and_walls.cracked_deepslate_bricks_slab": "裂纹深板岩砖台阶",
+  "block.more_slabs_stairs_and_walls.cracked_deepslate_bricks_stairs": "裂纹深板岩砖楼梯",
+  "block.more_slabs_stairs_and_walls.cracked_deepslate_bricks_wall": "裂纹深板岩砖墙",
+  "block.more_slabs_stairs_and_walls.cracked_deepslate_tiles_slab": "裂纹深板岩瓦台阶",
+  "block.more_slabs_stairs_and_walls.cracked_deepslate_tiles_stairs": "裂纹深板岩瓦楼梯",
+  "block.more_slabs_stairs_and_walls.cracked_deepslate_tiles_wall": "裂纹深板岩瓦墙",
+  "block.more_slabs_stairs_and_walls.cracked_nether_bricks_slab": "裂纹下界砖台阶",
+  "block.more_slabs_stairs_and_walls.cracked_nether_bricks_stairs": "裂纹下界砖楼梯",
+  "block.more_slabs_stairs_and_walls.cracked_nether_bricks_wall": "裂纹下界砖墙",
+  "block.more_slabs_stairs_and_walls.cracked_polished_blackstone_bricks_slab": "裂纹磨制黑石砖台阶",
+  "block.more_slabs_stairs_and_walls.cracked_polished_blackstone_bricks_stairs": "裂纹磨制黑石砖楼梯",
+  "block.more_slabs_stairs_and_walls.cracked_polished_blackstone_bricks_wall": "裂纹磨制黑石砖墙",
+  "block.more_slabs_stairs_and_walls.cracked_stone_bricks_slab": "裂纹石砖台阶",
+  "block.more_slabs_stairs_and_walls.cracked_stone_bricks_stairs": "裂纹石砖楼梯",
+  "block.more_slabs_stairs_and_walls.cracked_stone_bricks_wall": "裂纹石砖墙",
+  "block.more_slabs_stairs_and_walls.crimson_hyphae_slab": "绯红菌岩台阶",
```

```suggestion
  "block.more_slabs_stairs_and_walls.crimson_hyphae_slab": "绯红菌核台阶",
```
原文`Crimson Hyphae Slab`
全文通查